cmd/oci-runtime-tool/generate.go

```diff
@@ -21,77 +21,77 @@ var generateFlags = []cli.Flag{
+	cli.StringFlag{Name: "linux-seccomp-kill", Usage: "specifies syscalls to respond with kill"},
+	cli.BoolFlag{Name: "linux-seccomp-only", Usage: "specifies to export just a seccomp configuration file"},
+	cli.StringFlag{Name: "linux-seccomp-remove", Usage: "specifies syscalls to remove seccomp rules for"},
+	cli.BoolFlag{Name: "linux-seccomp-remove-all", Usage: "removes all syscall rules from seccomp configuration"},
+	cli.StringFlag{Name: "linux-seccomp-trace", Usage: "specifies syscalls to respond with trace"},
+	cli.StringFlag{Name: "linux-seccomp-trap", Usage: "specifies syscalls to respond with trap"},
+	cli.StringSliceFlag{Name: "linux-sysctl", Usage: "add sysctl settings e.g net.ipv4.forward=1"},
+	cli.StringSliceFlag{Name: "linux-uidmappings", Usage: "add UIDMappings e.g HostID:ContainerID:Size"},
+	cli.StringSliceFlag{Name: "mount-bind", Usage: "bind mount directories src:dest[:options...]"},
 	cli.StringFlag{Name: "mount-cgroups", Value: "no", Usage: "mount cgroups (rw,ro,no)"},
-	cli.StringFlag{Name: "mount-label", Usage: "selinux mount context label"},
-	cli.BoolFlag{Name: "no-new-privileges", Usage: "set no new privileges bit for the container process"},
-	cli.IntFlag{Name: "oom-score-adj", Usage: "oom_score_adj for the container"},
 	cli.StringFlag{Name: "os", Value: runtime.GOOS, Usage: "operating system the container is created for"},
 	cli.StringFlag{Name: "output", Usage: "output file (defaults to stdout)"},
-	cli.StringSliceFlag{Name: "poststart", Usage: "set command to run in poststart hooks"},
-	cli.StringSliceFlag{Name: "poststop", Usage: "set command to run in poststop hooks"},
-	cli.StringSliceFlag{Name: "prestart", Usage: "set command to run in prestart hooks"},
 	cli.BoolFlag{Name: "privileged", Usage: "enable privileged container settings"},
-	cli.StringSliceFlag{Name: "readonly-paths", Usage: "specifies paths readonly inside container"},
+	cli.StringSliceFlag{Name: "process-cap-add", Usage: "add Linux capabilities"},
+	cli.StringSliceFlag{Name: "process-cap-drop", Usage: "drop Linux capabilities"},
+	cli.StringFlag{Name: "process-cwd", Value: "/", Usage: "current working directory for the process"},
+	cli.IntFlag{Name: "process-gid", Usage: "gid for the process"},
+	cli.StringSliceFlag{Name: "process-groups", Usage: "supplementary groups for the process"},
+	cli.BoolFlag{Name: "process-no-new-privileges", Usage: "set no new privileges bit for the container process"},
+	cli.StringSliceFlag{Name: "process-rlimits-add", Usage: "specifies resource limits for processes inside the container. "},
+	cli.StringSliceFlag{Name: "process-rlimits-remove", Usage: "remove specified resource limits for processes inside the container. "},
+	cli.BoolFlag{Name: "process-rlimits-remove-all", Usage: "remove all resource limits for processes inside the container. "},
+	cli.StringFlag{Name: "process-selinux-label", Usage: "process selinux label"},
```

Comment:
`selinuxLabel` is part of the `process`, I think the `process-` will be better. `linux-selinux-label`may be confusing.